website/templates/project/project.mako

```diff
@@ -21,7 +21,7 @@
                     % endif
                 % endif
                 <h2 class="node-title">
-                    <span id="nodeTitleEditable" class="overflow">${node['title']}</span>
+                    <span id="nodeTitleEditable" class="overflow"> ${node['title']}</span>
```

Comment:
This slightly affects UI by prepending a space to the node's title. Please remove.